helm/regproc-camel/.gitignore

```diff
@@ -1,2 +1,2 @@
 charts/
-Charts.lock
+Chart.lock
```

Comment:
CHART_VERSION=1.3.0-beta.1-develop
SALTGEN_CHART_VERSION=1.3.0-beta.1-develop not changed